tracer/src/Datadog.Trace/RemoteConfigurationManagement/Transport/RemoteConfigurationApiFactory.cs

```diff
@@ -24,10 +24,7 @@ public static IRemoteConfigurationApi Create(ImmutableExporterSettings exporterS
                 () => new MinimalAgentHeaderHelper(),
                 uri => uri);
 
-            return
-                string.IsNullOrWhiteSpace(remoteConfigurationSettings.FilePath)
-                    ? RemoteConfigurationApi.Create(apiRequestFactory, discoveryService)
-                    : RemoteConfigurationFileApi.Create(remoteConfigurationSettings);
+            return RemoteConfigurationApi.Create(apiRequestFactory, discoveryService);
```

Comment:
I think you can remove entire class, and use direct call `RemoteConfigurationApi.Create(apiRequestFactory, discoveryService)`, instead?

Reply:
Could do yeah. I kind of like that it incapsulates this for now, as otherwise it's just more lines in the (already large) `Instrumentation` class. It could well make sense to remove it when you refactor the RCM creation into TracerManager though 🙂